service/controller/clusterapi/v17/key/cluster.go

```diff
@@ -1,14 +1,65 @@
 package key
 
 import (
+	"github.com/giantswarm/cluster-operator/pkg/label"
 	"github.com/giantswarm/microerror"
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 )
 
+func AWSClusterConfigName(cluster v1alpha1.Cluster) string {
+	return clusterProviderStatus(cluster).Cluster.ID
+}
+
+func ClusterAvailabilityZones(cluster v1alpha1.Cluster) []string {
+	azMap := make(map[string]struct{})
+
+	azMap[ClusterMasterAZ(cluster)] = struct{}{}
+
+	// TODO: Extract AZs from MachineDeployments
```

Comment:
Again with the nits but should this be a TODO issue?

Reply:
I don't think it's needed in this case because we need to figure the AZs in coming weeks anyway in order to get the cluster up & running with node pools (so it cannot get lost). :slightly_smiling_face:

Reply:
OK cool 👍